SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -546,6 +546,17 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 		return FPaths::Combine(SpatialGDKServicesConstants::SpatialOSSnapshotFolderPath, GetSpatialOSSnapshotToLoad());
 	}
 
+	FORCEINLINE FString GetProjectDefinedSchemaFolder() const
+	{
+		FString ProjectDefinedSpatialPath = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectDir(), TEXT("/../spatial")));
+		return FPaths::Combine(ProjectDefinedSpatialPath, TEXT("/custom_schema"));
```

Comment:
```suggestion
		return FPaths::Combine(SpatialGDKServicesConstants::SpatialOSDirectory, TEXT("/custom_schema"));
```
(Also means the line before it is unnecessary.)